packages/cli/README.md

```diff
@@ -173,7 +173,7 @@ Have a per-repository, self-hosted GraphQL Inspector service or deploy it with D
 
 ```bash
 # install
-yarn global add @graphql-inspector/actions
+pnpm global add @graphql-inspector/actions
```

Comment:
I am not sure it’s the same command for global installation

Reply:
`pnpm add --global @graphql-inspector/actions`